funfact/loss.py

```diff
@@ -99,3 +99,8 @@ class KLDivergence(Loss):
     '''KL Divergence loss.'''
     def _loss(self, model, target):
         return ab.multiply(target, ab.log(ab.divide(target, model)))
+
+
+mse = MSE()
```

Comment:
What is the reason for reintroducing these?

Reply:
In case if the user (or me in this case) wants to directly use a loss function as a *function*, without the need to instantiate it first. :grin: